src/core/api/instance_api.cpp

```diff
@@ -65,6 +65,17 @@ otInstance *otInstanceInitMultiple(uint8_t aIdx)
 
     return instance;
 }
+
+otInstance *otInstanceGetInstance(uint8_t aIdx)
+{
+    if (aIdx >= OPENTHREAD_CONFIG_MULTIPLE_INSTANCE_NUM)
+    {
+        return nullptr;
+    }
+
+    return &Instance::Get(aIdx);
```

Comment:
I know it seems convenient to use `rerturn nullptr` here, but per the style guide, we should have a single `return` from any method or function. Can you please help change this. 

It can be simplified to 
```c++
   return (aIdx >= OPENTHREAD_CONFIG_MULTIPLE_INSTANCE_NUM) ? nullptr : &Instance::Get(aIdx);`.
```